public/js/controllers/attackcost_controller.js

```diff
@@ -122,7 +122,8 @@ export default class extends Controller {
       'total', 'totalDCRPos', 'totalDeviceCost', 'totalElectricity', 'totalExtraCostRate', 'totalKwh',
       'totalPos', 'totalPow', 'graph', 'labels', 'projectedTicketPrice', 'projectedTicketPriceIncrease', 'attackType',
       'attackPosPercentAmountLabel', 'dcrPriceLabel', 'totalDCRPosLabel',
-      'projectedPriceDiv'
+      'projectedPriceDiv', 'attackNotPossibleWrapperDiv', 'coinSupply',
```

Comment:
remove the unused target `coinSupply`

Reply:
coinSupply target is used. Please check line 476.